it-and-security/default.yml

```diff
@@ -38,6 +38,11 @@ org_settings:
     zendesk: [ ]
   mdm:
     apple_bm_default_team: $DOGFOOD_APPLE_BM_DEFAULT_TEAM
+    end_user_authentication:
+      entity_id: dogfood-eula.fleetdm.com
+      idp_image_url: ""
+      idp_name: Google Workspace
+      metadata_url: $DOGFOOD_MDM_SSO_METADATA_URL
```

Comment:
this PR shouldn't be merged without setting this env variable first. I provided the value and asked for help in the #help-dogfooding channel in Slack

Reply:
currently tracked at https://github.com/fleetdm/confidential/issues/7926